app/src/main/kotlin/games/TicTacToe.kt

```diff
@@ -8,19 +8,20 @@ class TicTacToe(
         val player2: Player = Player(name = "Paul", id = 2)
 ) {
     private val greeting = "Welcome to the Tic Tac Toe game!"
-    private val board = Board()
+    internal val board = Board()
```

Comment:
we need to access this from within the tests, so we make it `internal` which seems to do the trick